internal/backend/runtime/omni/migration/migrations.go

```diff
@@ -1365,14 +1360,27 @@ func compressUncompressed[
 	T any,
 	S protobuf.Spec[T],
 ](update func(spec *protobuf.ResourceSpec[T, S]) (bool, error)) func(context.Context, state.State, *zap.Logger) error {
-	return func(ctx context.Context, st state.State, _ *zap.Logger) error {
+	return func(ctx context.Context, st state.State, l *zap.Logger) error {
 		items, err := safe.ReaderListAll[R](ctx, st)
 		if err != nil {
 			return err
 		}
 
+		if total := items.Len(); total > 0 {
+			md := items.Get(0).Metadata()
+
+			l = l.With(
+				zap.String("resource", md.String()),
+				zap.String("owner", md.Owner()),
+			)
+
+			l.Info("compressing resources", zap.Int("total_num", total))
+		}
+
 		for val := range items.All() {
 			if val.Metadata().Phase() != resource.PhaseRunning {
+				l.Info("skipping migration for resource in non-running phase")
```

Comment:
I think with 200k resources, these per-resource log lines would produce crazy amount of logs.

If we do want logs, here, I would rather do a summary line: `processed %d resources, %d skipped as already compressed, %d re-compressed, %d skipped as non-running`

Reply:
👍 https://github.com/siderolabs/omni/pull/972/files#diff-7a3b7babb5a5155c2301d949a7201d047f946d824771b60402cd3014c6b3096dR1409